Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,721 @@
+                    'Id': group['id'],
+                    'Policy': group['policy'],
+                    'PolicyVersion': group['policyVersion'],
+                    'SimulationStatus': group['endpointSettings'],
+                    'Endpoints': group['endpoints'],
+                    'CreationTime': group['creationTime']
+                }
+                context.append(group_context)
+            ec['Minerva.Group'] = context
+        elif context_id.endswith('vaccination'):
+            context = []
+            if isinstance(data, list):
+                for vaccine in data:
+                    vaccine_context = {
+                        'Id': vaccine['id'],
+                        'Name': vaccine['name'],
+                        'Type': vaccine['type'],
+                        'Description': vaccine['description'],
+                        "IsMonitorOnly": vaccine['isMonitorOnly'],
+                        'Last modified by': vaccine['lastModifiedBy'],
+                        'Last modified on': vaccine['lastModifiedOn']
+                    }
+                    context.append(vaccine_context)
+                ec['Minerva.Vaccine'] = context
+            else:
+                context = {
+                    'Id': data['id'],
+                    'Name': data['name'],
+                    'Type': data['type'],
+                    'Description': data['description'],
+                    "IsMonitorOnly": data['isMonitorOnly'],
+                    'Last modified by': data['lastModifiedBy'],
+                    'Last modified on': data['lastModifiedOn']
+                    }
+                ec['Minerva.Vaccine(val.Id === obj.Id)'] = context
+        elif context_id.endswith('exclusions'):
+            context = []
+            if isinstance(data, list):
+                for exclusion in data:
+                    exclusion_context = {
+                        'Id': exclusion['id'],
+                        'Type': exclusion['type'],
+                        'Exclusion data': exclusion['data'],
+                        'Description': exclusion['description'],
+                        'Last modified by': exclusion['lastModifiedBy'],
+                        'Last modified on': exclusion['lastModifiedOn'],
+                        'Applied groups': exclusion['appliedGroupsIds']
+                    }
+                    context.append(exclusion_context)
+                ec['Minerva.Exclusion'] = context
+            else:
+                context = {
+                    'Id': data['id'],
+                    'Type': data['type'],
+                    'Exclusion data': data['data'],
+                    'Description': data['description'],
+                    'Last modified by': data['lastModifiedBy'],
+                    'Last modified on': data['lastModifiedOn'],
+                    'Applied groups': data['appliedGroupsIds']
+                }
+                ec['Minerva.Exclusion(val.Id === obj.Id)'] = context
```

Comment:
Do we need the val.Id == obj.Id?

Reply:
Sure, as for the case the data retrieved is not a list, we'd like to remove duplicates (as you wrote me in the email)

Reply:
That applies also to lists, it should be in all context entries. Does it not work when you add it to the other entries?

Reply:
my bad, omitted the context ids. pushed the fix